src/app/components/Recommendations/RecommendationsItem/index.tsx

```diff
@@ -0,0 +1,46 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { useContext } from 'react';
+
+import { RequestContext } from '#contexts/RequestContext';
+import useClickTrackerHandler from '#app/hooks/useClickTrackerHandler';
+import { Recommendation } from '#app/models/types/onwardJourney';
+import RecommendationsImage from '../RecommendationsPromoImage';
+import styles from './index.styles';
+
+const eventTrackingData = {
+  componentName: 'midarticle-mostread',
+};
+
+const RecommendationsItem = ({
+  recommendation,
+}: {
+  recommendation: Recommendation | null;
+}) => {
+  const { isLite } = useContext(RequestContext);
+
+  const handleClickTracking = useClickTrackerHandler(eventTrackingData);
+
+  if (!recommendation) return null;
```

Comment:
Let's move this up to line 20 so that we don't call any unnecessary code

Reply:
Can't render hooks conditionally 😬

Reply:
Is this a hook though?